spec/unit/pushprocessor.spec.ts

```diff
@@ -198,11 +198,12 @@ describe("NotificationService", function () {
     });
 
     it("should add default rules in the correct order", () => {
-        matrixClient.pushRules = PushProcessor.rewriteDefaultRules({
+        const pushRules = PushProcessor.rewriteDefaultRules({
```

Comment:
ok this makes things clearer, but still, if we're not going to use any of the stuff in the `beforeEach`, let's move it to a different `describe` block. It's really confusing at the moment.